### PR TITLE
MetaDraw for crosslinked peptide annotation

### PR DESCRIPTION
**Pull request by lonelu**
TsvResultReader can be less restricted in the future.

### CODE REVIEW

File: GUI/MetaDraw.xaml.cs [hunk 1]
```diff
@@ -155,7 +156,7 @@ private void DrawPsm(int oneBasedScanNumber, string fullSequence = null)
             mainViewModel.DrawPeptideSpectralMatch(msDataScanToDraw, psmToDraw);
 
             // draw annotated base sequence
-            DrawAnnotatedBaseSequence(psmToDraw);
```
Comment (rmillikin) [inline GUI/MetaDraw.xaml.cs]: is this available for XL yet? maybe check to see if the PSM is a crosslink, and draw it if it isn't?

**Comment by rmillikin** [comment]
looks very nice 👍 just need that one change to the base sequence annotation

File: GUI/MetaDraw.xaml.cs (hunk 1 above)
Comment (lonelu) [reply GUI/MetaDraw.xaml.cs]: Ready to merge.
The DrawAnnotatedBaseSequence will be changed in the future for glyco or crosslink peptide.

**Comment by rmillikin** [comment]
GitHub is giving me a weird error trying to update the branch from master... I'll try again later today/tmw. PR looks good 👍

**Pull request merged**